CMSgov/mint-app
# [EASI-4291] - Map other type translations

# EASI-4291


## Changes and Description

- Changes Possible Operation Need/Solution names - (removed 'possible')
- Updated 'other' types `parentReferenceLabel` where applicable per figma
- Added `exportLabel` to labels to all that render `Select all that apply`
- Ran export translation
- Updated snapshots



## How to test this change

Verify changes in translations match figma changes
https://www.figma.com/file/5Ux715qVJHg38EbJKavMR4/MINT---Change-History?type=design&node-id=771-33209&mode=design&t=pTpdIRVkHdXB8O6f-0

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
